ldclient/polling.py

```diff
@@ -0,0 +1,34 @@
+from threading import Thread
+
+from ldclient.interfaces import UpdateProcessor
+from ldclient.util import log
+import time
+
+
+class PollingUpdateProcessor(Thread, UpdateProcessor):
+    def __init__(self, api_key, config, requester, store):
+        Thread.__init__(self)
+        self.daemon = True
+        self._api_key = api_key
+        self._config = config
+        self._requester = requester
+        self._store = store
+        self._running = False
```

Comment:
I don't *think* there are any implications of this, and I notice we use the same pattern in the `StreamProcessor`, but this doesn't look thread safe to me.